### DISCUSSION TITLE
Using Vue Components as Pages like MD files

### DISCUSSION THREAD

**Discussion by Abiti-936** [vuejs/vitepress#898] [General]
I'm thinking about using Vue Components as Pages rather than MD files. What do you think about it?

**Comment by WebMechanic** [comment]
I'd better noch answer this, however, what'd be the rationale of this?

**Comment by inca** [comment]
Please add support for this. VitePress is absolutely great for SSG, but not all statically generated sites are docs. And markdown is only good for writing docs and blog posts.

If you have a very simple landing page, or a gallery, or product portfolio, or a simple ecommerce, or anything in between — you have to use md files as container/entrypoint for your vue files. And if you actually try to compose pages from reusable Vue files + HTML templates, then you'll quickly find Markdown interference pretty annoying (e.g. a single blank line rule to delimit the blocks, or indented blocks automatically becoming code blocks, etc, etc).
